Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Lighthouse"
 uuid = "ac2c24cd-07f0-4848-96b2-1b82c3ea0e59"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.14.10"
+version = "0.14.11"
```

Comment:
On Monday, let's pick up our convo w/ @ericphanson about whether this should be a breaking release OR if we should bump the schema versions of `ClassRow`.

- [ ] Block merge on decision